classes/battle/BattleManager.php

```diff
@@ -830,9 +875,13 @@ protected function applyAttack(BattleAttack $attack, Fighter $user, Fighter $tar
         }
 
         if($attack->jutsu->hasEffect()){
-            $text .= "<p style=\"font-style:italic;margin-top:3px;\">" .
-                $this->system->db->clean($this->effects->getAnnouncementText($attack->jutsu->effect)) .
-                "</p>";
+            foreach ($attack->jutsu->effects as $effect) {
+                if ($effect && $effect->effect != 'none') {
```

Comment:
Actually I wonder about this, should we even have an effect in the array that has the value of "none" or should we just not have any values in the array if the jutsu has no effects?